MQTicketService/startApp.sh

```diff
@@ -20,4 +20,4 @@ sleep 30
 # Set environment
 . /opt/mqm/bin/setmqenv -s
 # Execute TicketGenerator
-/opt/mqm/java/jre64/jre/bin/java -cp ./jarFiles/*: -Djava.library.path=/opt/mqm/java/lib64 com.ibm.mq.badge.Manager
+/jdk-11.0.13+8-jre/bin/java -cp /data/TicketGenerator/jarFiles/*: -Djava.library.path=/opt/mqm/java/lib64 com.ibm.mq.badge.Manager
```

Comment:
With a maven build inside the docker image, the run command should get updated to run the uber jar.

Reply:
Changed the shell script command as required.